server/internal/controllers/users.go

```diff
@@ -273,28 +273,34 @@ func UserPurge(c *gin.Context) {
 		return
 	}
 
-	ok, _, _, _ := auth.AuthenticateUserOfChain(c, db, " ", query.UserUID)
+	ok, user, _, _ := auth.AuthenticateUserOfChain(c, db, "", query.UserUID)
```

Comment:
The second result of `AuthenticateUserOfChain` is the user related to `query.UserUID`

An empty `""` is very different to `" "`